openrtb_ext/bidders.go

```diff
@@ -147,6 +147,7 @@ const (
 	BidderInvibes           BidderName = "invibes"
 	BidderIQZone            BidderName = "iqzone"
 	BidderIx                BidderName = "ix"
+	BidderJanet             BidderName = "janet"
```

Comment:
I see the name is written as "JANet" in other places. You are welcomed to use this casing for the bidder name variable, ie: `BidderJANet BidderName = "janet"`. This is important because it changes my reading of the name from "janet" to "j-a net". The bidder code and file names are appropriate as `janet`.

Reply:
I too would welcome the changing of the casing for the bidder name variable!